contracts/Coordinator.sol

```diff
@@ -79,11 +82,18 @@ contract Coordinator is ICoordinator {
     function withdrawCollateralUnderNFT(uint256 amount, uint256 nftId) external override notImplementedYet {}
 
     function borrowUnderNFT(uint256 _nftId, uint256 _amount) external override {
-        IERC20(_tokenLvUSD).transfer(_tokenVaultOUSD, _amount);
+        IERC20(_tokenLvUSD).transfer(_tokenExchanger, _amount);
         CDPosition(_tokenCDP).borrowLvUSDFromPosition(_nftId, _amount);
     }
 
-    function repayUnderNFT(uint256 _nftId, uint256 _amount) external override notImplementedYet {}
+    function repayUnderNFT(uint256 _nftId, uint256 _amount) external override {
```

Comment:
It worth adding a comment explaining the parameters
what's "_amount"? is it lvUSD? OUSD? USDT?

Reply:
Good point, will add